README.md

```diff
@@ -38,17 +38,17 @@ The EyeWire REST API provides programmatic access to interact with EyeWire's dat
 
 # EyeWire Structure & Terminology
 ## The Dataset
-The EyeWire dataset comprises pairs of .jpg from which we can map a 3D volume. EyeWire calls these images a `channel` and a `segmentation`. The images are grouped into `chunks`, which are then grouped into `volumes`.
+The EyeWire dataset comprises pairs of .jpg and .png images from which we can map a 3D volume. EyeWire calls the sets of .jpg images `channels` and the sets of .png images `segmentations`. The sets are grouped into `chunks`, which are then grouped into `volumes`.
 
 - `channel`
-    - an 8bpp greyscale .jpg file
+    - a set of 8bpp greyscale .jpg files
     - the electron microscope images
 - `segmentation`
-    - a .png file
+    - a set of 24bpp RGB .png files for normal `tasks` or 32bpp RGBA .png files for omni `tasks`
     - used to describe which pixels belong to which `segment` ID
-    - The RGBA color represents the `segment` ID: R + 256*G + 256^2 * B + 256^3 * A
-  `voxel`
-      the highest precision of the dataset. Can be considered a 1x1x1 pixel cube.
+    - The RGBA color represents the `segment` ID: R + 256*G + 256^2 * B + 256^3 * A. The alpha channel is excluded in non-omni `tasks`.
```

Comment:
I think we should remove references to omni or non omni. For now, the api is focusing on non-omni (eyewire). I would change the equation to just `R + 256*G + 256^2 * B` which is the equation I use in the sample app.